modal_proto/api.proto

```diff
@@ -261,8 +261,17 @@ message AppStats {
   string environment_name = 11;
 }
 
+enum AppSortType {
```

Comment:
We try to follow this [style guide](https://buf.build/docs/best-practices/style-guide#enums) for protobuf naming, and so according to it should leave `0` reserved for "UNSPECIFIED" and also prefix all the fields with `APP_SORT_`.